test/assert.js

```diff
@@ -182,6 +182,110 @@ test('.throws()', function (t) {
 	t.end();
 });
 
+test('.throws() returning Error', function (t) {
+	t.doesNotThrow(function () {
+		var assertion = assert.throws(function () {
+			throw new Error('foo');
+		});
+
+		assert.same('foo', assertion.message);
```

Comment:
I just went ahead and killed off the whole bunch of the specs - just the one test now to verify `actual === expected`. Regardless of secondary arguments passed to `throws`, it should always be the same. sound good to you @novemberborn?